.buildkite/docs.definition.yaml

```diff
@@ -0,0 +1,14 @@
+agent_queue_id: trigger-pipelines
+description: GDK for Unity Docs Pipeline
+github:
+  branch_configuration: []
+  default_branch: docs-current
+  pull_request_branch_filter_configuration:
+    - "docs-current"
+    - "docs-next"
+    - "docs/*"
+teams:
+- name: Everyone
```

Comment:
Do you want everyone to be able to trigger the builds?

Reply:
it's now `READ_ONLY`